pkgs/development/tools/pre-commit/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, pythonPackages, pew }:
+with pythonPackages; buildPythonApplication rec {
+  name = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.

Reply:
Yeah, that's redundant. fixed it